src/Umbraco.Web.UI.Client/src/common/directives/components/editor/umbbreadcrumbs.directive.js

```diff
@@ -58,6 +58,28 @@ Use this directive to generate a list of breadcrumbs.
                 }
             };
 
+            scope.openPath = function (ancestor, event) {
+                // targeting a new tab/window?
+                if (event.ctrlKey || 
+                    event.shiftKey ||
+                    event.metaKey || // apple
+                    (event.button && event.button === 1) // middle click, >IE9 + everyone else
+                ) {
+                    // yes, let the link open itself
+                    return;
+                }
+                event.stopPropagation();
+                event.preventDefault();
+
+                var path = scope.pathTo(ancestor);
+                $location.path(path);
+                navigationService.clearSearch(["cculture"]);
```

Comment:
@kjac i think i'm confused but need to double check here, i though the purpose of this PR is to not automatically retain cculture but for the breadcrumb links we would retain it but this seems to be explicitly clearing it?

Reply:
Hehe it does look weird, but the input array contains the parameters that should *not* be cleared from search. Yeah...

Reply:
ahh, haha... i should read my own APIs, lol, that clears all except for that :) all good

Reply:
Basically clearSearch clears everything but the passed keys.